_posts/2016-09-10-akka-stream-kafka.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: series_post
+title: "Akka Streams Kafka 0.11"
+description: ""
+author: Adam Warski
+category: integrations
+series_title: Integration
+series_tag: integration
+tags: [streams,integration]
+---
+{% include JB/setup %}
+
+
+[Apache Kafka](http://kafka.apache.org) is the leading distributed messaging system, and [Reactive Streams](http://www.reactive-streams.org) is an emerging standard for asynchronous stream processing. It seems natural to combine these two; that’s why [SoftwareMill](https://softwaremill.com) started the [reactive-kafka](https://github.com/akka/reactive-kafka) project back in December 2014 and maintained it since.
```

Comment:
Would it not be good to explain that apps often use Kafka WITH Akka in the apps?
I often get confused people who ask "Kafka vs. Akka"-like questions.